### ISSUE TITLE
Dropdown - Single selection - VoiceOver doesn't narrate button label

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14280]
Environment:
chrome "84.0.4147.105"
macos 10.15.5

Open the production URL:
https://fluentsite.z22.web.core.windows.net/components/dropdown/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#types-dropdown
2. Navigate into "Selection" example
3. Navigates to the menu button
4. observe narration 

Actual:
VoiceOver narrates only "button"
<img width="1440" alt="Screenshot 2020-07-31 at 5 37 22 PM" src="https://user-images.githubusercontent.com/22620150/89051836-1e2a7000-d355-11ea-8f49-89a2fcd9abd4.png">

Expected:
label of the button is narrated

Comment:
looks like the patter for labeling somehow doesn't works with VoiceOver, works fine with NVDA and JAWS
maybe we should add only aria-label on the button, but then it need be tested properly

**Comment by paulgildea** [comment]
👍
